astronomer/airflow/version_check/update_checks.py

```diff
@@ -342,8 +342,7 @@ def available_update(self):
             if "runtime" in _app_name().lower():
                 rel_parsed_base_version = rel_parsed_version.major
                 base_version = ac_version.major
-
-            if rel_parsed_version > ac_version and rel_parsed_base_version == base_version:
+            if rel_parsed_version > ac_version and rel_parsed_base_version >= base_version:
```

Comment:
It happened that the test was failing because the latest version (6.0.0) was not shown. The comparison here was wrong because it compares that base version must be equal. It should be greater or equal e.g 6 >= 5 would return the latest version.

Reply:
Wrong. This needs separate handling of runtime/AC

Reply:
Ok. It was the test that was wrong. The test was to test image on highest patch release but was using runtime version `5.0.0`. It was passing before because the highest patch version of 5.0 series was 5.0.8 and that was also the highest image version. Now we have both 5.0.9 and 6.0.0 so it was requesting for 6.0.0 which is what is expected.

I have used 4.2.6 as the runtime version on the image since 4.0 series are no longer released. This tests that once we are at the highest level of a patch release, then we show the user the latest release.